src/test/jtx/Env.h

```diff
@@ -147,6 +149,11 @@ class Env
         memoize(Account::master);
         Pathfinder::initPathTable();
         construct(std::forward<Args>(args)...);
```

Comment:
1. I can't think of any reason not to have it on. Right now my one test that disables it was really just there to prove that disabling by config actually works.
2. good point - let's see what @JoelKatz says about the config use case and then I'll possibly change the comment and/or the test case.